webui/src/components/OKR.vue

```diff
@@ -109,6 +115,24 @@
                 this.logChange(`Me deleted '${objId}'`);
             },
 
+            sendChangeSuggestion(objective) {
+                const targetSubject = this.$store.state.user.selectedSubject;
+                window.location = `mailto:${targetSubject.mail || targetSubject.userPrincipalName}?\
```

Comment:
why upn if no mail - https://support.cloudhq.net/it-admin-office-365-why-your-user-principal-name-upn-should-match-your-email-address/